### PR TITLE
Vault Compensation Reference Price Date Poll, initial draft

### PR DESCRIPTION
**Pull request by jpritikin**
Please discuss

### CODE REVIEW

**Comment by jpritikin** [comment]
@OperationNine Thank you for your review.

File: governance/polls/Vault Compensation Reference Price Date - May 11, 2020.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: Vault Compensation Reference Price Date Poll - May 11, 2020
+summary: Signal which date to use for ETH/BAT references prices vs DAI for vault compensation
+discussion_link: https://forum.makerdao.com/t/2275
+poll_rules: The voter may rank one or more of the poll options or they may elect to abstain from the poll entirely
```
Comment (jparklev) [inline governance/polls/Vault Compensation Reference Price Date - May 11, 2020.md:5]:
```suggestion
vote_type: Ranked Choice IRV
```
updated metadata tag 🙂  (assuming ranked choice is what you want)

File: governance/polls/Vault Compensation Reference Price Date - May 11, 2020.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+options:
```
Comment (jparklev) [inline governance/polls/Vault Compensation Reference Price Date - May 11, 2020.md:6]: would you be amenable to making the options something like 
```md
   0: Abstain
   1: A
   2: B
   3: C
```
and putting the description for each in the document body? The current UI won't display the length of option `3` very well. (using letters is just a gentle suggestion, any change that shortens what's displayed in the dropdown would work.)

Comment (jpritikin) [reply governance/polls/Vault Compensation Reference Price Date - May 11, 2020.md:6]: Yeah, do you want to make the change? The short names could be `M13`, `Cur`, and `Mean(M13, Cur)`?

Comment (jpritikin) [reply governance/polls/Vault Compensation Reference Price Date - May 11, 2020.md:6]: OK, I pushed a commit for the abbreviations. Does this look okay?

Comment (jparklev) [reply governance/polls/Vault Compensation Reference Price Date - May 11, 2020.md:6]: Nice, yea -- looks good to me 👍

**Pull request closed without merging**